Xit/Preferences/PreviewsPrefsPane.swift

```diff
@@ -86,6 +89,28 @@ struct PreviewsPrefsPane: View
           }
         } label: { EmptyView() }.fixedSize()
       })
+      LabeledField("Commit view defaults:", VStack(alignment: .leading) {
```

Comment:
A couple of comments about this:
- These settings aren't for file previews, so I think they should go in the General pane
- Labeling it "commit view defaults" implies that you can change these settings for individual windows, which isn't the case, so I think it should just be "commit view"
- Instead of packing the checkbox and number field into a single line, I think it would be better to separate them into two lines

Reply:
done!